### PR TITLE
Update vpc-vpn-bastion.yaml

### PR DESCRIPTION
**Pull request by markotitel**
Add UDP port 40000. vpnclient uses this port for UDP acceleration.
Mobile clients work fine, but connecting from Linux using default client settings, connection will "freeze".
Once port 40000 is allowed in security group, browsing is perfect.
https://github.com/SoftEtherVPN/SoftEtherVPN/issues/544

### CODE REVIEW

File: vpc/vpc-vpn-bastion.yaml [hunk 1]
```diff
@@ -295,6 +295,10 @@ Resources:
         IpProtocol: tcp
         FromPort: 5555
         ToPort: 5555
+      - CidrIp: 0.0.0.0/0
+        IpProtocol: udp
+        FromPort: 40000
+        ToPort: 40000
```
Comment (michaelwittig) [inline vpc/vpc-vpn-bastion.yaml]: If I get this https://github.com/SoftEtherVPN/SoftEtherVPN/blob/1e17c9bcfd7e7b31756aa5389bcbff76c2c9c88a/src/Cedar/UdpAccel.h#L139 right we have to set `ToPort` to `44999`

**Comment by markotitel** [comment]
I just see that I've made this pull request like a true amateur which I am.
And do not want to spam with additional PRs.
How should I proceed?

File: vpc/vpc-vpn-bastion.yaml (hunk 1 above)
Comment (markotitel) [reply vpc/vpc-vpn-bastion.yaml]: You are right. I've debuged actuall issue using tcpdump.
Should I close this PR and create a new one?

**Comment by michaelwittig** [comment]
No need to worry. Thanks for spotting this issue!

Are you familiar with updating an existing pull request? If you change the `ToPort ` attribute to `44999 ` in https://github.com/markotitel/aws-cf-templates this PR will be updated and I can merge it.

**Comment by markotitel** [comment]
I hope I've done it right. I thouth updating PR will "trash" the commits history. I've just added one more commit and see the PR updated, if this is again wrong. I will need to read a bit more on Github pull requests.

**Comment by michaelwittig** [comment]
Yes, you did. I squashed your two commits into one! Thanks for investigating the issue and fixing it! Your fix will be included in the following release.

**Pull request merged**